src/pages/edit_resource/description_step/TagsField.tsx

```diff
@@ -71,6 +72,14 @@ const TagsField: FC<TagsFieldProps> = ({ setAllChangesSaved }) => {
     setAllChangesSaved(false);
     try {
       const promiseArray: Promise<any>[] = [];
+      const cleanTagArray: string[] = [];
+      tagArray.forEach((tag) => {
+        tag = tag.replaceAll('#', '');
+        if (cleanTagArray.indexOf(tag) === -1) {
+          cleanTagArray.push(tag);
```

Comment:
Har dere mulighet til å sette en boolean på at den har vasket bort #? Og så senere vise med rød tekst at den har blitt vasket? De spørsmålstegn boksene tror jeg ingen desverre leser (men gjerne la teksten i spørsmålstegn boksen stå)

Reply:
<img width="532" alt="tags" src="https://user-images.githubusercontent.com/44363838/143199958-ab8a49eb-3646-45cd-b5eb-a8fda75b9927.PNG">

Reply:
Forøvrig: sett den røde teksten under input feltet. (bedre for skjermlesere)